app/.env.example

```
@@ -36,6 +36,17 @@ VITE_SKILLS_GITHUB_REPO=tinyhumansai/openhuman-skills
 # Only anonymous page views and feature-engagement events are sent — no PII.
 VITE_GA_MEASUREMENT_ID=
 
+# [optional] OpenPanel analytics project. Leave client id blank to disable.
+# The API URL should include `/api` and must accept this browser/client id.
+VITE_OPENPANEL_CLIENT_ID=e9c996d5-497f-4eec-9bde-630019ad525b
+VITE_OPENPANEL_API_URL=https://panel.tinyhumans.ai/api
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_OPENPANEL_API_URL key should go before the VITE_OPENPANEL_CLIENT_ID key

(UnorderedKey)